boards/arm/nrf52840_mdk/board.cmake

```diff
@@ -2,3 +2,4 @@
 
 board_runner_args(pyocd "--target=nrf52840")
 include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
```

Comment:
Are you sure about this? Why not use openocd-nrf5.board.cmake instead? That would get you thread-aware debugging for free.

Reply:
Ah, I see, it's a different interface. Never mind. No objections to this patch, although you could probably adapt openocd-nrf5.board.cmake to get thread aware debugging with your interface as well. see the `pre_init_cmds` variable set in that file for details.